pkg/testcase/node.go

```diff
@@ -52,3 +52,108 @@ func TestNodeStatus(
 	_, err := shared.GetNodes(true)
 	Expect(err).NotTo(HaveOccurred())
 }
+
+func TestKillall() {
+	product, err := shared.Product()
+	Expect(err).NotTo(HaveOccurred())
+
+	g := GinkgoT()
+	cluster := factory.ClusterConfig(g)
+
+	for i := len(cluster.ServerIPs) - 1; i > 0; i-- {
+
+		cmd := fmt.Sprintf("sudo %s-killall.sh", product)
+		_, err := shared.RunCommandOnNode(cmd, cluster.ServerIPs[i])
+		Expect(err).NotTo(HaveOccurred())
+	}
+
+	switch {
+	case product == "k3s":
+		res, _ := shared.RunCommandHost("kubectl get nodes --kubeconfig=" + shared.KubeConfigFile)
+		Expect(res).To(ContainSubstring("refused"))
+	case product == "rke2":
+		res, _ := shared.RunCommandHost("kubectl get nodes --kubeconfig=" + shared.KubeConfigFile)
+		Expect(res).To(SatisfyAny(ContainSubstring("timed out"), ContainSubstring("refused")))
+	default:
+		shared.ReturnLogError("unsupported product: %s\n", product)
+	}
+}
+
+func StopServer() {
```

Comment:
stop server is a test or a aux function ?

Reply:
also for this we already have a function on shared.aux - product
func serviceName(product, nodeType string) (string, error) {